stdlib/public/SwiftShims/swift/shims/CMakeLists.txt

```diff
@@ -34,11 +34,9 @@ add_custom_command(
     OUTPUT "${output_dir}"
     COMMAND ${CMAKE_COMMAND} "-E" "make_directory" "${output_dir}")
 
-if(SWIFT_BUILD_STATIC_STDLIB)
-  add_custom_command(
-      OUTPUT "${output_dir_static}"
-      COMMAND ${CMAKE_COMMAND} "-E" "make_directory" "${output_dir_static}")
-endif()
+add_custom_command(
```

Comment:
Is this meant to be completely uncovered, or should this and the non-static case be guarded on whether or not a static build is requested, or whether it's very specifically the Linux-static SDK build (sort of like the stdlib CLibdispatch build up above)?

Reply:
Having it uncovered is pretty harmless; it does mean we generate an extra `swift_static` directory that wasn't there before, containing the shims and a symlink to the clang headers. I _guess_ we could loop over all of the configured SDKs to see if any of them are static-only, which would avoid doing that if we haven't been asked to do any kind of static stdlib build.